src/onboarding.ts

```diff
@@ -101,11 +101,11 @@ export class Onboarding extends Logger {
     if (!password) {
       throw new Error('password is a required parameter!');
     }
-
-    const runtimeConfig: any = await Onboarding.generateRuntimeConfig(
-      mnemonic, password, runtime.web3,
-    );
-    const runtimeNew = await createDefaultRuntime(runtime.web3, runtime.dfs, runtimeConfig);
+    const runtimeNew = await createDefaultRuntime(runtime.web3, runtime.dfs, {
```

Comment:
`New` is always bit general, maybe we have a more semantic wording, like `identityRuntime`? I'm not sure what it does here ...

Reply:
It's the runtime for the given mnemonic and password. We could rename it to onboardingRuntime, but it's obvious in this context :D